app/scraper/sources/face_book.py

```diff
@@ -23,15 +23,43 @@
     MAJOR_FULL_NAMES = json.load(f)
 
 
+# Define a function to check if S3 credentials are set
+def has_s3_credentials():
+    return os.environ.get("S3_ACCESS_KEY") and os.environ.get("S3_SECRET_ACCESS_KEY")
+
+
+# Define a dummy ImageUploader that doesn't perform any actions related to S3
+
+
+class DummyImageUploader:
```

Comment:
I feel that this might be better placed alongside the actual ImageUploader. A better solution might be to just make the ImageUploader itself function as DummyImageUploader when the requisite credentials aren't available

Reply:
Better to have them be separate because there are other things that need to change outside of self.image_uploader, including whether to initialize self.fernet

Reply:
Fernet isn't related to S3. Is there anything else that needs to change?